server/src/main/java/org/opensearch/index/IndexModule.java

```diff
@@ -165,6 +168,47 @@ public final class IndexModule {
         Property.NodeScope
     );
 
+    public static List<String> INDEX_STORE_HYBRID_ALL_EXTENSIONS = List.of(
```

Comment:
This constant probably belongs to FsDirectoryFactory (and should be private, that's the only place it is used)

Reply:
Fixed it